packages/components/src/ComboBox/docs/ComboBoxOption/autoMenuWidth.tsx

```diff
@@ -1,25 +1,24 @@
-import { ComboBox, ComboBoxOption, ComboBoxOptions, Label, Text } from "@hopper-ui/components";
+import { ComboBox, ComboBoxItem, Text } from "@hopper-ui/components";
 
 export default function Example() {
     return (
         <ComboBox
             aria-label="list of options with a description"
+            isAutoMenuWidth
+            label="Roles"
         >
-            <Label>Roles</Label>
-            <ComboBoxOptions isAutoMenuWidth>
-                <ComboBoxOption textValue="Developer">
-                    <Text>Developer</Text>
-                    <Text slot="description">Builds, tests, and maintains software.</Text>
-                </ComboBoxOption>
-                <ComboBoxOption textValue="Cat">
-                    <Text>Designer</Text>
-                    <Text slot="description">Creates visually appealing, functional solutions.</Text>
-                </ComboBoxOption>
-                <ComboBoxOption textValue="Frog">
-                    <Text>Manager</Text>
-                    <Text slot="description">Responsible for leading and overseeing a team or department to ensure organizational goals are met.</Text>
-                </ComboBoxOption>
-            </ComboBoxOptions>
+            <ComboBoxItem textValue="Developer">
+                <Text>Developer</Text>
+                <Text slot="description">Builds, tests, and maintains software.</Text>
+            </ComboBoxItem>
+            <ComboBoxItem textValue="Cat">
```

Comment:
Just wondering if the textValue shouldn't be Designer and Manager down there?

Reply:
It was probably previously missed. I'll fix it.